packages/hint/src/bin/hint.ts

```diff
@@ -12,33 +12,22 @@
  * ------------------------------------------------------------------------------
  */
 
-const tracking = (/--tracking[=\s]+([^\s]*)/i).exec(process.argv.join(' '));
+const telemetry = (/--tracking[=\s]+([^\s]*)/i).exec(process.argv.join(' '));
 
-import { configStore, appInsights } from '@hint/utils';
+import { appInsights } from '@hint/utils';
 
-const trackingEnv = process.env.HINT_TRACKING;
-let enableTracking;
+const telemetryEnv = process.env.HINT_TRACKING;
```

Comment:
Should we change also the env variable?

Reply:
There's an issue for that (#2995). I don't want to make a breaking change right now.

Reply:
ok!